src/plothist/__init__.py

```diff
@@ -1,6 +1,6 @@
 """Plot histograms in a scalable way and a beautiful style."""
 
-__version__ = "1.1.2"
+__version__ = "1.2.0"
```

Comment:
In the past we were doing `1.X` not `1.X.0`. I don't mind if we change now.

Reply:
Let's change then, easier to find+replace